crates/node/p2p/src/net/driver.rs

```diff
@@ -78,21 +78,21 @@ impl Network {
                 }
                 select! {
                     event = self.gossip.select_next_some() => {
-                        trace!(target: "p2p::driver", "Received event: {:?}", event);
+                        trace!("Received event: {:?}", event);
```

Comment:
```suggestion
                        trace!(target: "p2p::driver",
                            %event,
                            "Received event"
                        );
```
I would do it like this. i would highly recommend to keep the targets, that way you can zoom in on certain components when debugging, e.g. `RUST_LOG=p2p::driver,p2p::discv5=trace`

Reply:
You can still do this natively using the module as a target
<img width="1731" alt="Screenshot 2025-04-07 at 8 07 57 PM" src="https://github.com/user-attachments/assets/ce5b0a34-d28d-449f-b32b-9e5857d8d92d" />